tests/test_threadpoolctl.py

```diff
@@ -373,7 +375,7 @@ def test_libomp_libiomp_warning(recwarn):
     from ._openmp_test_helper import check_nested_openmp_loops  # noqa
 
     # Trigger the import of numpy to potentially import Intel OpenMP via MKL
-    import numpy.linalg  # noqa
+    pytest.importorskip("numpy.linalg")
```

Comment:
there's no way we can reach this part if numpy is not present since the test is skipped if scipy is not present.

Reply:
There is no dependency on scipy for this one. `check_nested_openmp_loops` has no dependency on numpy/scipy.

Reply:
right, I mixed 2 tests...